### PR TITLE
feat: add stacks APIs to codegen

### PR DESCRIPTION
**Pull request by danxmoran**
Part of #18

Had to override everything again here to split up the `schemas` and tweak the tags on the routes. I made #162 to track revisiting all the overrides after we've pushed the initial release.

### CODE REVIEW

File: api/contract/overrides/paths/stacks_stack_id.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  tags:
+    - Stacks
+  summary: Retrieve a stack
+  parameters:
+    - in: path
+      name: stack_id
+      required: true
+      schema:
+        type: string
+      description: Theidentifier of the stack.
```
Comment (wbaker85) [inline api/contract/overrides/paths/stacks_stack_id.yml:12]: Missing a space!

Comment (wbaker85) [reply api/contract/overrides/paths/stacks_stack_id.yml:12]: Some below are as well

**Review by wbaker85 — APPROVED** [review]
LGTM!

**Comment by danxmoran** [comment]
Oops, clicked merge before processing the comments... follow-up PR incoming!

**Pull request merged**